File: 2019/territoriali/antivirus/statement/antivirus.pas
const MAXN = 1000;

procedure solve(test: longint);
var
    N1, N2, N3, N4: longint;
    M: longint;
    F1, F2, F3, F4: ansistring;

    (* scrivi in queste variabili la risposta *)
    p1, p2, p3, p4: longint;

begin

    readln(N1, N2, N3, N4);
    readln(M);
    readln(F1);
    readln(F2);
    readln(F3);
    readln(F4);

    (* RISOLVI QUI IL PROBLEMA!! *)

    writeln('Case #', test, ': ', p1, ' ', p2, ' ', p3, ' ', p4);
end;


var
    T, test: longint;

begin
    (* le seguenti due righe possono essere utili per leggere e scrivere da file *)

    (*
    assign(input, 'input.txt'); reset(input);
    assign(output, 'output.txt'); rewrite(output);
    *)

    readln(T);

    for test := 1 to T do
        solve(test);
end.

File: 2019/territoriali/antivirus/statement/antivirus.c
#include <stdio.h>

#define MAXN 1000

void solve(int t) {
    int N1, N2, N3, N4;
    int M;
    char F1[MAXN+1], F2[MAXN+1], F3[MAXN+1], F4[MAXN+1];

    // scrivi in queste variabili la risposta
    int p1, p2, p3, p4;

    scanf("%d %d %d %d", &N1, &N2, &N3, &N4);
    scanf("%d", &M);
    scanf("%s %s %s %s", F1, F2, F3, F4);

    // RISOLVI QUI IL PROBLEMA!!

    printf("Case #%d: %d %d %d %d\n", t, p1, p2, p3, p4);
}

int main() {
    // le seguenti due righe possono essere utili per leggere e scrivere da file

    //freopen("input.txt", "r", stdin);
    //freopen("output.txt", "w", stdout);

    int T, t;
    scanf("%d", &T);

    for (t = 1; t <= T; t++) {
        solve(t);
    }
}
